dotnet/src/Skills/Skills.MsGraph/Connectors/MicrosoftGraphModelExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using Microsoft.Graph;
+using Microsoft.Graph.Extensions;
+using Microsoft.SemanticKernel.Skills.MsGraph.Models;
+
+namespace Microsoft.SemanticKernel.Skills.MsGraph.Connectors;
+
+/// <summary>
+/// Extensions for converting between Microsoft Graph models and skill models.
+/// </summary>
+internal static class MicrosoftGraphModelExtensions
+{
+    /// <summary>
+    /// Convert a Microsoft Graph message to an email message.
+    /// </summary>
+    public static Models.EmailMessage ToEmailMessage(this Message graphMessage)
```

Comment:
You don't want to do a _using Microsoft.SemanticKernel.Skills.Models_ at the top of the file?

Reply:
The Graph SDK collides HARD with the names, so I wanted to be very specific in here.